spec/observables/combineLatest-spec.js

```diff
@@ -3,15 +3,69 @@ var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.combineLatest', function () {
+  function concat(x, y) { return '' + x + y; }
+
+  it('should be never when sources are never and never', function () {
```

Comment:
You are right! I didn't know this file.. I'll close this PR and take a look in this file.